pulpcore/pulpcore/app/serializers/task.py

```diff
@@ -79,7 +79,8 @@ class TaskSerializer(ModelSerializer):
     created_resources = CreatedResourceSerializer(
         help_text=_('Resources created by this task.'),
         many=True,
-        read_only=True
+        read_only=True,
+        view_name='NotActuallyUsed'
```

Comment:
What about calling this `'None`' instead?

Reply:
Also can a brief (1-2 lines maybe) comment be added about how this is a polymorphic field so we don't know the view name.